elementary/monitor/api/alerts/normalized_alert.py

```diff
@@ -71,20 +72,20 @@ def _normalize_alert(self):
             )
             return self.alert
 
-    def _get_alert_subscribers(self) -> List[Optional[str]]:
-        subscribers = []
-        test_subscribers = self.test_meta.get(SUBSCRIBERS_KEY, [])
-        model_subscribers = self.model_meta.get(SUBSCRIBERS_KEY, [])
-        if isinstance(test_subscribers, list):
-            subscribers.extend(test_subscribers)
+    def _get_alert_members(self, meta_key: str) -> List[Optional[str]]:
```

Comment:
Maybe a better name will be `_get_alert_attribute` or `_get_alert_config`?
Members is a bit confusing